docs/features/colorbutton/README.md

```diff
@@ -72,6 +72,30 @@ CKEditor will then output the color definition as `<font>` elements with `color`
 	<p><font color="#800080">This is my text color.</font><br/>
 	<font style="background-color:#FFFF00;">This is my background color</font></p>
 
+Since CKEditor 4.15.0 it is possible to use `class` instead of `style` to set more advanced styles to text.
```

Comment:
Well, yes and no, you could use classes before too. But since the changes in what `color` value outputs in `4.12` it produced invalid class names :see\_no\_evil: 

So I would start by saying that in `4.15` we have introduced `colorName` which uses color name instead of color code, which allows to create descriptive class names.